packages/bpk-component-graphic-promo/src/BpkGraphicPromo.module.scss

```diff
@@ -0,0 +1,256 @@
+  background-position: center;
+  background-size: cover;
+  box-shadow: none;
+
+  @include breakpoint-landscape {
+    pointer-events: none;
+
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-small();
+
+      @include bpk-label-1;
+    }
+
+    &__sponsor-content {
+      margin-bottom: bpk-spacing-xl();
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-label-1;
+    }
+
+    &__strapline {
+      @include bpk-body-default;
+    }
+
+    &__headline {
+      @include breakpoint-landscape-desktop {
+        @include bpk-hero-5;
+      }
+
+      @include breakpoint-landscape-tablet {
+        @include bpk-heading-1;
+      }
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
+    }
+  }
+
+  @include breakpoint-portrait-large {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-4;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-subheading;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-hero-4;
+    }
+
+    &__strapline {
+      @include bpk-subheading;
+    }
+  }
+
+  @include breakpoint-portrait-medium {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-3;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-subheading;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-hero-5;
+    }
+
+    &__strapline {
+      @include bpk-body-longform;
+    }
+  }
+
+  @include breakpoint-portrait-small {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-sm();
+
+      @include bpk-label-1;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-label-1;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-2;
+    }
+
+    &__strapline {
+      @include bpk-body-default;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+
+    @include breakpoint-landscape {
+      justify-content: center;
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+      }
+
+      &--end {
+        align-items: end;
+      }
+    }
+
+    @include breakpoint-portrait {
+      justify-content: space-between;
+
+      &--inverted {
+        flex-direction: column-reverse;
+      }
+    }
+
+    @include breakpoint-landscape-desktop {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+    }
+
+    @include breakpoint-landscape-tablet {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xl(), false);
+    }
+
+    @include breakpoint-portrait-large {
+      height: pxtorem(933);
```

Comment:
Are we able to just define the rem value instead of doing a conversion?

Might also be worth seeing if we can make this up based on existing spacing values or at least creating a local variable to define what the number is?

As looking at this it just looks like a magic number and might cause confusion to future contributors or maintainers.

If you need the maths to do px to rem conversion its pixel value / 16 = rem value